packages/discord-types/src/stores/UserStore.d.ts

```diff
@@ -0,0 +1,11 @@
+import { FluxStore, User } from "..";
+
+export class UserStore extends FluxStore {
+    filter(filter: () => boolean, sort?: boolean): Record<string, User>;
+    findByTag(username: string, discriminator: string): User;
+    forEach(action: Function): void;
```

Comment:
not even typed properly 😭 
```suggestion
    filter(filter: (user: User) => boolean, sort?: boolean): Record<string, User>;
    findByTag(username: string, discriminator: string): User;
    forEach(action: (user: User) => void): void;
```